source/pkg/adapter/adapter.go

```diff
@@ -108,7 +108,7 @@ func (a *Adapter) Start(ctx context.Context) error {
 
 	a.logger.Info("Number of consumers from config:", zap.Int("NumConsumers", a.config.NumConsumers))
 
-	for i := 0; i < a.config.NumConsumers; i++ {
+	for i := 0; i < a.config.NumConsumers; i++ { //TODO: Read NumConsumers data from new config struct??
```

Comment:
@lionelvillard now I have a new [config.go setup ](https://github.com/knative-sandbox/eventing-redis/pull/10/files#diff-0812a6fafa0498638fcd9e77c3514dbcf642c696bc2f963726e197767eebeefb)for config-redis CM. But how do I read the[ data key/value](https://github.com/knative-sandbox/eventing-redis/pull/10/files#diff-0812a6fafa0498638fcd9e77c3514dbcf642c696bc2f963726e197767eebeefbR32) from that new config into the adapter config?

Reply:
usually it's done by passing the config as environment variables.